<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>handelOrders</class>
 <widget class="QDialog" name="handelOrders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>31</y>
     <width>171</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="OrderNr">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>240</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ShowOrder">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Visa Order</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Visa Order</string>
   </property>
  </widget>
  <widget class="QLabel" name="OrderNummerLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Order Nummer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Rensa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createOrder">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Skapa ny order</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Goback">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Gå tillbaka</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>212</width>
     <height>105</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Order nummer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>IdNr</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Antal varor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="orderNr2"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="idNr"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="nrOfItems">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="addOrderLine">
      <property name="text">
       <string>Lägg till Order rad</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
